### PR TITLE
Relying Party Spelling

### PR DESCRIPTION
**Pull request by dimitribouniol**
I noticed that the term "relying party" was sometimes spelled out, sometimes not. This PR consistently spells it out, by renaming the following:
- `PublicKeyCredentialRequestOptions.rpId` to `PublicKeyCredentialRequestOptions.relyingPartyID`
- `PublicKeyCredentialRpEntity` to `PublicKeyCredentialRelyingPartyEntity`
- remaining local variables that used `rp` to `relyingParty`

### CODE REVIEW

**Comment by marius-se** [comment]
The other renames are fine!

File: Sources/WebAuthn/Ceremonies/Authentication/PublicKeyCredentialRequestOptions.swift [hunk 1]
```diff
@@ -29,7 +29,7 @@ public struct PublicKeyCredentialRequestOptions: Encodable {
     public let timeout: UInt32?
 
     /// The Relying Party ID.
-    public let rpId: String?
```
[the remark this answers is not in the document]
Comment (dimitribouniol) [reply Sources/WebAuthn/Ceremonies/Authentication/PublicKeyCredentialRequestOptions.swift:32]: Can this be solved by pointing out the differentiation in the doc comment? Nice thing about Codable is if folks use the structure as is, they'll get the right naming on the other side, and I really don't like the idea of making the Swift API more cryptic for the sole reason of matching JS naming standards 😅

[the remark this answers is not in the document]
Comment (dimitribouniol) [reply Sources/WebAuthn/Ceremonies/Authentication/PublicKeyCredentialRequestOptions.swift:32]: Also, we do some similar renaming for aesthetic reasons here:
https://github.com/swift-server/webauthn-swift/blob/95ce09430d13852db971db56ae0b55d1911db2ee/Sources/WebAuthn/Ceremonies/Registration/PublicKeyCredentialCreationOptions.swift#L56-L63

[the remark this answers is not in the document]
Comment (0xTim) [reply Sources/WebAuthn/Ceremonies/Authentication/PublicKeyCredentialRequestOptions.swift:32]: I think a comment is fine here - explicit naming is usually recommended by the Swift API guidelines

**Review by 0xTim — APPROVED** [review]
I'm happy with this - @marius-se what do you think? Feel free to merge if ok

**Comment by 0xTim** [comment]
@swift-server-bot please test this

**Comment by 0xTim** [comment]
@marius-se feel free to open an issue for discussion if you feel strongly, but I'm going to merge this as I'm trying to clear through my GH issues 😆

**Pull request merged**